Formula/pgsql-ogr-fdw.rb

```diff
@@ -11,8 +11,8 @@ def pour_bottle?
     false
   end
 
-  depends_on "postgis"
-  depends_on "gdal"
+  depends_on "osgeo/homebrew-osgeo4mac/postgis"
+  depends_on "osgeo/homebrew-osgeo4mac/gdal2"
```

Comment:
@PeterPetrik since the new postgis formula pulls in gdal2, you don't need to specify gdal2 here, it will get handled transitively. That's most likely the reason why the the Travis build is failing.

Reply:
let me try, thanks